docs/topics/coroutines-basics.md

```diff
@@ -245,35 +245,39 @@ Done
 
 <!--- TEST -->
 
-## Coroutines ARE light-weight
+## Coroutines are light-weight
 
-Run the following code:
+Coroutines are less resource-intensive than JVM threads. Code that exhausts the
+JVM's available memory when using threads can be expressed using coroutines
+without hitting resource limits. For example, the following code launches
+100000 distinct coroutines that each wait 5 seconds and then print a period
+('.'):
 
 ```kotlin
 import kotlinx.coroutines.*
 
-//sampleStart
 fun main() = runBlocking {
+//sampleStart
     repeat(100_000) { // launch a lot of coroutines
         launch {
             delay(5000L)
             print(".")
         }
     }
-}
 //sampleEnd
+}
```

Comment:
Given that the code is not runnable, I think a better choice would be to remove `//sampleStart` and `//sampleEnd`, given how they aren't recognized for code that is not runnable: https://kotlinlang.org/docs/coroutines-basics.html#coroutines-are-light-weight